docs/configuration.rst

```diff
@@ -0,0 +1,233 @@
+Configuration options
+#####################
+
+The Raven client can be configured through a bunch of configuration options.
+Some of them are automatically populated by environment variables when a new
+instance of the ``Configuration`` class is created. The following is an exhaustive
+list of them with a description of what they do and some example values.
+
+Send attempts
+=============
+
+The number of attempts that should should be made to send an event before erroring
+and dropping it from the queue.
+
+.. code-block:: php
+
+    $configuration = new Configuration(['send_attempts' => 1]);
+    $configuration->setSendAttempts(1);
+
+By default this option is set to 6.
+
+Server
+======
+
+The DSN of the Sentry server the authenticated user is bound to.
+
+.. code-block:: php
+
+    // Before Sentry 1.9 the private DSN must be used
```

Comment:
Sentry 9 is the version where this is deprectated 😉 (1.9 mentioned a few more times)